hashing/main.cpp

```diff
@@ -152,8 +157,14 @@ static int example(void)
 }
 
 int main() {
-    int ret = example();
+    mbedtls_platform_context platform_ctx;
```

Comment:
Question: How large can these structs be? I think that the benchmark example used to have constant problems with stack size. Is this just going to contribute to making the problem worse?

Reply:
Agreed, but that is my point: that PR is not merged, so depending on the size of mbedtls_platform_context and what features are enabled by the config there might be stack issues.

Reply:
@andresag01 I pointed to that PR, to make a push on its merging:)